app/src/release/res/values/google_maps_api.xml

```diff
@@ -0,0 +1,22 @@
+<resources>
```

Comment:
No, it is a auto-generated file in the release folder. The other one is in src/main.